.github/workflows/lra-test.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: LRA test
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+  pull_request:
+    branches: [ master ]
+    paths:
+      - "rts/lra/**"
+
+jobs:
+  lra-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Narayana - build
```

Comment:
I think we should change this and the subsequent lines to be something like
`PROFILE=LRA ./scripts/hudson/narayana.sh`

But also, we need to rebase the PR first:
`./scripts/hudson/narayana-rebase.sh`

Reply:
@tomjenkinson without clean rebase the GH actions won't run. Please check the example at https://github.com/xstefank/narayana/pull/5. So I don't think narayana-rebase.sh is necessary.

About `PROFILE=LRA ./scripts/hudson/narayana.sh` this job was created precisely to test in different environment with different settings so for instance avoiding `-Dlra.tck.timeout.factor=1.5` from this script. Also GH actions will set PR status directly on PR so no need for comments. It is easier to keep only the command for now. If we are to move the whole CI from Hudson here then it will make sense to reuse the scripts with adjustments. But now it would be just overhead. Sorry, if I'm missing something in the scripts. In that case please point me to the correct configuration and I'll change it in the PR. But now I would like to avoid setting a bunch of environment properties to run a single script that runs essentially single command.

Reply:
Do I understand that this will rebase before running the testing somehow because of GH Actions?

The narayana-rebase.sh actually does a rebase so we can be sure the PR is running with the latest changes.

If that is the case then we don't need the narayana-rebase.sh

Reply:
It won't run the actions before you solve all the conflicts. So it basically forces you to rebase. It's true that clean rebase without conflicts could help with many merges in short periods but that will rarely happen I think and we will still be running hudson jobs this is just parallel testing for the timing issues. It won't be easy to set GH actions according to your hudson setup. This would be just a start and if there is a requirement to migrate everything, including scripts, we can do that later in a separate JBTM. 

The scripts require some environment properties that aren't configurable per run in GH actions.

Reply:
The automatica rebasing of PRs has a purpose in that if an unrelated defect is fixed and merged, it doesn't require the PR author to rebase manually and the project can just re-run the PR.

Reply:
GH actions won't run if the PR is not updated. So it wouldn't have a chance to rebase automatically.

Reply:
I have edited my comment which may have made it clearer what I was getting at before but if not then I wasn't thinking the job would ever run automatically, I more meant that someone from the project team can just re-rerun it. But yes, I understand from you now that it might not be possible to re-run tests when using GitHub actions until there is an update in the code (unless GitHub actions can respond to comments from maintainers maybe?)